@JackBoosY Hmm that doesn't work @ f596542f6:
```sh
$ gh pr checkout 19287
$ ./bootstrap-vcpkg.sh
$ ./vcpkg install tinyfiledialogs
```

Then in another dir with this above it:
```cmake
cmake_minimum_required(VERSION 3.4)
cmake_policy(SET CMP0048 NEW)
project(pp VERSION 0.0.0 LANGUAGES C)

file(DOWNLOAD "https://sourceforge.net/p/tinyfiledialogs/code/ci/4b65621952f2cf1045837a6380aa912b028daa47/tree/hello.c?format=raw"
              "src/main.c"
     EXPECTED_HASH SHA256=72502b4b5e93bf262b1ff148de028d12a187c1fda45de89212a3f896e2be1827)
set(src "${CMAKE_CURRENT_BINARY_DIR}/src/main.c")
file(READ "${src}" _c)
# Also tried `<tinyfiledialogs/tinyfiledialogs.h>`
string(REPLACE [[#include "tinyfiledialogs.h"]] [[#include <tinyfiledialogs.h>]] _c "${_c}")
file(WRITE "${src}" "${_c}")

find_package(tinyfiledialogs CONFIG REQUIRED)

add_executable("${PROJECT_NAME}" "${src}")

target_link_libraries("${PROJECT_NAME}" PRIVATE tinyfiledialogs::tinyfiledialogs)
```

```sh
$ cmake -DCMAKE_BUILD_TYPE='Debug' -DCMAKE_TOOLCHAIN_FILE='[path]/vcpkg/scripts/buildsystems/vcpkg.cmake' ..
$ cmake --build .
build/src/main.c:54:10: fatal error: 'tinyfiledialogs.h' file not found
#include <tinyfiledialogs.h>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
gmake[2]: *** [CMakeFiles/pp.dir/build.make:76: CMakeFiles/pp.dir/src/main.c.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/pp.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2
```

--- a/ports/tinyfiledialogs/CMakeLists.txt
+++ b/ports/tinyfiledialogs/CMakeLists.txt
@@ -3,6 +3,8 @@ project(tinyfiledialogs C)
 
 add_library(tinyfiledialogs tinyfiledialogs.c)
 
+target_include_directories(tinyfiledialogs INTERFACE $<INSTALL_INTERFACE:include/tinyfiledialogs>)
+
 install(
     TARGETS tinyfiledialogs
     EXPORT tinyfiledialogsConfig

--- a/versions/t-/tinyfiledialogs.json
+++ b/versions/t-/tinyfiledialogs.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "02f1f2a70350a2bf3701cf6279af2ab577d0e6da",
+      "git-tree": "3959a47c7d93ca7db6e2022553b1d3427970cecf",
       "version-semver": "3.8.8",
       "port-version": 1
     },

